Detectors/TPC/base/include/TPCBase/CDBInterface.h

```diff
@@ -236,6 +241,25 @@ inline T& CDBInterface::getObjectFromCDB(std::string_view path)
   return *object;
 }
 
+/// Get a CalPad object stored in templated formats from the CCDB.
+/// @tparam T
+/// @param path
+/// @param timestamp
+/// @param metaData
+/// @return The object from the CCDB, ownership is transferred to the caller.
+/// @todo Consider removing in favour of calling directly the manager::get method.
+template <typename T>
+inline T& CDBInterface::getSpecificObjectFromCDB(std::string_view path, long timestamp, std::map<std::string, std::string> metaData)
+{
+  static auto& cdb = o2::ccdb::BasicCCDBManager::instance();
+  auto* object = cdb.getSpecific<T>(path.data(), timestamp, metaData);
+  return *object;
+}
+
+template CalPad& CDBInterface::getSpecificObjectFromCDB(const std::string_view path, long timestamp, std::map<std::string, std::string> metaData);
+template std::vector<CalPad>& CDBInterface::getSpecificObjectFromCDB(const std::string_view path, long timestamp, std::map<std::string, std::string> metaData);
+template std::unordered_map<std::string, o2::tpc::CalPad>& CDBInterface::getSpecificObjectFromCDB(const std::string_view path, long timestamp, std::map<std::string, std::string> metaData);
```

Comment:
I guess the argument should be `const std::map<std::string, std::string>& metaData` ?

Reply:
Good point, I changed it. The failing build seems to be unrelated as far as I can see, though.